### ISSUE TITLE
Touch screen compatibility

### ISSUE THREAD

**Issue by hypocube** [OpenRCT2/OpenRCT2#2091]
Could a checkbox be added to the options menu that changes the drag function to left click? I'm playing rct2 on a touchscreen (it's a lot of fun) and the only drawback right now is I need the map to move around effectively.

**Comment by IntelOrca** [comment]
It would be better to hook onto the touch down message instead in the message loop.
I already implemented pinch zooming awhile back.

**Comment by hawkerm** [comment]
Yeah, I'd be nice to just be able to drag on the screen with touch to pan around.  Would make it really nice to play on a touchscreen.
